includes/widgets/playing-cards.php

```diff
@@ -0,0 +1,710 @@
+
+	private static $suit_char_map = [
+		'diamond' => '&#9830;',
+		'heart' => '&#9829;',
+		'club' => '&#9827;',
+		'spade' => '&#9824;',
+	];
+
+	private static $value_map = [
+		'1' => 'A',
+		'2' => 2,
+		'3' => 3,
+		'4' => 4,
+		'5' => 5,
+		'6' => 6,
+		'7' => 7,
+		'8' => 8,
+		'9' => 9,
+		'10' => 10,
+		'11' => 'J',
+		'12' => 'Q',
+		'13' => 'K',
+	];
+
+	public function get_name() {
+		return 'playing-cards';
+	}
+
+	public function get_title() {
+		return esc_html__( 'Playing Cards', 'elementor' );
+	}
+
+	public function get_icon() {
+		return 'eicon-heart';
+	}
+
+	public function get_keywords() {
+		return [ 'playing', 'cards' ];
+	}
+
+	private function register_cards_section_controls() {
+		$this->start_controls_section(
+			'cards_section',
+			[
+				'label' => esc_html__( 'Items', 'elementor' ),
+				'tab' => \Elementor\Controls_Manager::TAB_LAYOUT,
+			]
+		);
+
+		$repeater = new \Elementor\Repeater();
+
+		$repeater->add_control(
+			'card_title',
+			[
+				'label' => esc_html__( 'Title', 'elementor' ),
+				'type' => \Elementor\Controls_Manager::TEXT,
+				'default' => esc_html__( 'My Card', 'elementor' ),
+				'label_block' => true,
+			]
+		);
+
+		$repeater->add_control(
+			'card_value',
+			[
+				'label' => esc_html__( 'Card Value', 'elementor' ),
+				'type' => \Elementor\Controls_Manager::SELECT,
+				'default' => '1',
+				'options' => static::$value_map,
+			]
+		);
+		$repeater->add_control(
+			'card_suit',
+			[
+				'label' => esc_html__( 'Card Suit', 'elementor' ),
+				'type' => \Elementor\Controls_Manager::SELECT,
+				'default' => 'diamond',
+				'options' => static::$suit_char_map,
+			]
+		);
+		$this->add_control(
+			'list',
+			[
+				'label' => esc_html__( 'Card List', 'elementor' ),
+				'type' => \Elementor\Controls_Manager::REPEATER,
+				'fields' => $repeater->get_controls(),
+				'default' => [
+					[
+						'card_title' => esc_html__( 'Card #1', 'elementor' ),
+						'card_suit' => 'diamond',
+						'card_value' => '1',
+					],
+					[
+						'card_title' => esc_html__( 'Card #2', 'elementor' ),
+					],
+				],
+				'title_field' => '{{{ card_title }}}',
+			]
+		);
+
+		$this->end_controls_section();
+	}
+
+	private function register_layout_section_controls() {
+		$this->start_controls_section(
+			'layout_section',
+			[
+				'label' => esc_html__( 'Layout', 'elementor' ),
+				'tab' => \Elementor\Controls_Manager::TAB_LAYOUT,
+			]
+		);
+
+		$this->add_responsive_control(
+			'content_width',
+			[
+				'label' => esc_html__( 'Card Width', 'elementor' ),
+				'type' => \Elementor\Controls_Manager::SLIDER,
+				'size_units' => [ 'px', '%', 'em', 'rem', 'custom' ],
+				'range' => [
+					'px' => [
+						'min' => 160,
+						'max' => 1000,
+						'step' => 1,
+					],
+					'%' => [
+						'min' => 0,
+						'max' => 100,
+					],
+				],
+				'selectors' => [
+					'{{WRAPPER}} .e-playing-cards-container' => '--e-card-width: {{SIZE}}{{UNIT}};',
```

Comment:
```suggestion
					'{{WRAPPER}}' => '--e-card-width: {{SIZE}}{{UNIT}};',
```

Reply:
If you wrap the css variables with the widget class (see above), then you can simplify the control selectors.